.github/workflows/push-to-master.yml

```diff
@@ -1,8 +1,5 @@
 name: push-to-master
-on:
-  push:
-    branches:
-      - master
+on: pull_request
```

Comment:
Will be returned back after changes are approved (`on` parameter should be set to `pull_request` value to ensure that `test docs` job from `push-to-master.yml` config is fixed).